src/main/java/sirius/web/http/Response.java

```diff
@@ -108,7 +110,7 @@
      * Contains a set of parameter names which are censored in any output as we do not want to log user passwords etc.
      */
     private static final Set<String> CENSORED_LOWERCASE_PARAMETER_NAMES =
-            ImmutableSet.of("password", "passphrase", "secret", "secretKey");
+            Collections.unmodifiableSet(new HashSet<>(Arrays.asList("password", "passphrase", "secret", "secretKey")));
```

Comment:
Gut dass da Java in den nächsten Version auch was gescheites gebaut hat, ist ja übel mit den 3 geschachtelten Sets/Arrays :D

Reply:
yep, Java9 to the rescue...